packages/landing/src/components/common/header/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import HeaderLink from './HeaderLink'
+
+export default function Header() {
+  return (
+    <header className="container flex flex-row py-4 mb-6 lg:mb-12">
+      <HeaderLink href="/" withRightMargin>
+        DevHub
+      </HeaderLink>
+
+      <div className="flex-1" />
+
+      <HeaderLink href="/pricing" withRightMargin>
+        Pricing
+      </HeaderLink>
+
+      <HeaderLink href="/pricing" withRightMargin>
```

Comment:
@brunolemos Shouldn't this be `/changelog` instead of `/pricing`? Copy and paste error?

Reply:
yeah but I didn’t make these pages yet so it’s fine